src/Ledger/Prelude.agda

```diff
@@ -4,6 +4,7 @@ module Ledger.Prelude where
 
 open import Prelude public
 
+open import Data.These
```

Comment:
This doesn't do anything without `public`, and you might want to move this into `Prelude`